src/BuildProcess/BuildContainerImage.php

```diff
@@ -58,4 +67,46 @@ protected function getTagName()
     {
         return Manifest::name().':'.$this->environment;
     }
+
+    /**
+     * Ensure the provided Dockerfile is compatible with the runtime.
+     *
+     * @param  string  $environment
+     * @return null|void
+     */
+    protected function validateDockerFile($environment)
```

Comment:
Sorry, we need to tests on this. Do you mind of adding a couple of tests?

Reply:
This is pretty difficult to test in its current form as there is no way to swap out the Docker build. Didn't really want to make wholesale changes to that, but will see if I can find a way.